@navikt/core/react/src/modal/Modal.tsx

```diff
@@ -99,6 +97,10 @@ export const Modal = forwardRef<HTMLDialogElement, ModalProps>(
     }: ModalProps,
     ref,
   ) => {
+    const { cn } = useRenameCSS();
+
+    const polyfillClassName = useRef(cn("navds-modal--polyfilled"));
```

Comment:
Er det behov for å bruke `useRef` her?

Reply:
Siden den er statisk og ikke skal endres gjør useRef at man ikke trenger å legge den til som dependency i denne snippeten
```
useEffect(() => {
  if (needPolyfill && modalRef.current && portalNode) {
    dialogPolyfill.registerDialog(modalRef.current);
    modalRef.current.classList.add(polyfillClassName.current);
  }
  if (modalRef.current && portalNode) modalRef.current.autofocus = true;
}, [portalNode]);
```